circle.yml

```diff
@@ -0,0 +1,13 @@
+machine:
+  ruby:
+    version: 2.2.2
+  environment:
+    ROOT_URL: https://act.sumofus.org/
+    ROOT_ACTION_URL: https://act.sumofus.org/a/
+    AK_HOST: https://action.sumofus.org/rest/v1/
+    AK_USERNAME: ak_username
+    AK_PASSWORD: ak_password
+    ENGLISH_URI: /rest/v1/language/100/
+    GERMAN_URI: /rest/v1/language/101/
+    SPANISH_URI: /rest/v1/language/102/
+    FRENCH_URI: /rest/v1/language/103/
```

Comment:
> more interestingly though - why do we have these language IDs in the config?

They're required for page creation; you can't create a page through the API without including a valid URI for the language for the page. Unfortunately, AK's logic behind them is kind of annoying - they aren't (as far as I know) tied to anything in AK. We created all those languages and so each organization's languages are going to be distinct, so the ENV seemed like the right place to load those in, like other variable config values. 

The current logic around them probably isn't fantastic, but I went YAGNI for this and just made it simply load the values that applied to our organization.